packages/access-client/src/agent-data.js

```diff
@@ -171,18 +171,19 @@ export const isSessionProof = (delegation) =>
  * Get a map from CIDs to the session proofs that reference them
  *
  * @param {AgentData} data
- * @returns {Record<string, Ucanto.Delegation>}
+ * @returns {Record<string, [Ucanto.Delegation, ...Ucanto.Delegation[]]>}
  */
 export function getSessionProofs(data) {
-  /** @type {Record<string, Ucanto.Delegation>} */
+  /** @type {Record<string, [Ucanto.Delegation, ...Ucanto.Delegation[]]>} */
```

Comment:
This is ok but I would prefer `Record<Ucanto.DID, Record<string, ...Ucanto.Delegation>>` per https://github.com/web3-storage/w3up/pull/1047#discussion_r1378213049 because it makes it a lot easier to pick the ones you care and throw away others as opposed to having to go and group them in another loop.

Not a big deal, but unless there is compelling reason to prefer this I would suggest reverse.

Reply:
@Gozala roger. I will change to that.

I assume the keys of the inner record (typed as `string`) would be session proof issuer? and keys of outer record would be the attested to proof cid?

Reply:
did it https://github.com/web3-storage/w3up/pull/1047/commits/914405d21f57cb7c0969c767930e47c0dfa06e46